app/javascript/angularapps/openpicker/controllers/fetched_image_preview_controller.js

```diff
@@ -0,0 +1,54 @@
+ FetchedImagePreviewController = function($scope, $timeout, $http, DataAccessService, $sce, $filter, $location, Upload, OptionsService, LogService) {
```

Comment:
Done. All views , controllers and code related to preview is now called `FilePreview`